src/Sentry/Sentry.csproj

```
@@ -21,6 +21,17 @@
   </ItemGroup>
   <!-- Ben.Demystifier -->
 
+  <!-- DiagnosticSource -->
+  <PropertyGroup Condition="!$(TargetFramework.StartsWith('netstandard')) and '$(TargetFramework)' != 'net461'">
+    <DefineConstants>$(DefineConstants);HAS_DIAGNOSTIC_INTEGRATION</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'net461' and !$(TargetFramework.StartsWith('netstandard'))">
```

Keep the order consistent with the node above?

```suggestion
  <ItemGroup Condition="!$(TargetFramework.StartsWith('netstandard')) and '$(TargetFramework)' != 'net461'">
```